activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+*   Add expiry and purpose to MessageEncryptor & MessageVerifier.
```

Comment:
Also let's make it verifier first, encryptor second throughout the changelog since signing is less complex than encrypting (which for the MessageEncryptor is really encrypt and sign).